app/src/main/res/layout/fragment_create_shop.xml

```diff
@@ -0,0 +1,251 @@
+            android:orientation="vertical"
+            >
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="10dp"
+                android:gravity="center"
+                android:text="@string/create_new_shop"
+                android:textColor="@color/color_orange_mos"
+                android:textSize="@dimen/sp_30"
+                />
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_margin="@dimen/dp_10"
+                android:background="@drawable/boder_title"
+                android:orientation="vertical"
+                android:padding="@dimen/dp_10"
+
+                >
+                <android.support.design.widget.TextInputLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:paddingTop="@dimen/dp_10"
+                    android:textColorHint="@color/color_black"
+                    app:hintTextAppearance="@color/color_black"
+                    >
+                    <android.support.design.widget.TextInputEditText
+                        android:id="@+id/text_title_device"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:hint="@string/shop_name"
+                        android:textSize="@dimen/sp_14"
+                        />
+                </android.support.design.widget.TextInputLayout>
+
+                <android.support.design.widget.TextInputLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/dp_10"
+                    android:paddingTop="@dimen/dp_10"
+                    android:textColorHint="@color/color_black"
+                    app:hintTextAppearance="@color/color_black"
+                    >
+                    <android.support.design.widget.TextInputEditText
+                        android:id="@+id/text_description_device"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:hint="@string/description"
+                        android:textSize="@dimen/sp_14"
+                        />
+                </android.support.design.widget.TextInputLayout>
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    >
+                    <TextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginStart="@dimen/dp_2"
```

Comment:
ERROR: [RtlCompat] To support older versions than API 17 (project specifies 14) you should *also* add `android:layout_marginLeft="@dimen/dp_2"`
 API 17 adds a `textAlignment` attribute to specify text alignment. However, if you are supporting older versions than API 17, you must *also* specify a gravity or layout_gravity attribute, since older platforms will ignore the `textAlignment` attribute.